content/en/developers/new_tools/add_tool.md

```diff
@@ -155,7 +155,10 @@ If the container doesn't work yet, it's sometimes useful to try and troubleshoot
 16) Edit an entry for your package in your fork of neurocommand/blob/main/neurodesk/apps.json based on one of the other entries (generating one menu item for opening a terminal inside the containers, and one menu item for the GUI, if relevant). Notice that in the json file, the version field should contain the _buildDate_
 17) Include an icon file in your fork of neurocommand/neurodesk/icons
 18) Send a pull request from your fork of neurocommand to https://github.com/NeuroDesk/neurocommand/ 
-19) When the pull request is merged by Neurodesk admins, it will trigger an action to build the singularity container, distribute it in all object storage locations and on CVMFS, and it will update the menus in the desktop image on the next daily build. The new menu entry will be available after 24 hours in neurodesktop:latest and will be announced publicly with the next release.
-20) Check in the dev build if everything is ok before releasing a new version of Neurodesktop:
-<pre class="language-shell command-line" data-prompt="$"><code>sudo docker pull vnmd/neurodesktop-dev:latest && sudo docker run   --shm-size=1gb -it --privileged --name neurodesktop   -v ~/neurodesktop-storage:/neurodesktop-storage   -e HOST_UID="$(id -u)" -e HOST_GID="$(id -g)"   -p 8080:8080 -h neurodesktop-dev   vnmd/neurodesktop-dev:latest</code></pre>
-21) Consider contributing a tutorial about the new tool: https://github.com/NeuroDesk/neurodesk.github.io/tree/hugo-docsy/content/en/tutorials
+19) When the pull request is merged by Neurodesk admins, it will trigger an action to build the singularity container, distribute it in all object storage locations and on CVMFS, and it will update the menus in the desktop image on the next daily build. 
+20) Wait at least 24 hours
+21) Download and run the daily build of neurodesktop to check that your app can be launched from the start menu and works properly:
+<pre class="language-shell command-line" data-prompt="$"><code>sudo docker pull vnmd/neurodesktop:latest && sudo docker run   --shm-size=1gb -it --privileged --name neurodesktop   -v ~/neurodesktop-storage:/neurodesktop-storage   -e HOST_UID="$(id -u)" -e HOST_GID="$(id -g)"   -p 8080:8080 -h neurodesktop-latest   vnmd/neurodesktop:latest</code></pre>
+22) Post an issue in the open an issue in https://github.com/NeuroDesk/neurocontainers/issues notifying that your app appears in the start menu and tested. The app will be included in the next release of Neurodesktop, and will be mentioned in the public announcement that accompanies the release. If the app is not in the start menu or not working as expected based on your earlier testing, open an issue as well, and report it.
```

Comment:
typo: Post an issue in the open an issue ->

Reply:
I committed a correction